Style Guide and Diataxis Review: Streams III (#225)
iotaledger/streams: documentation/docs/libraries/wasm/examples.md

- javascript
- node
- examples

> - reference

Reviewer: woudn't the examples fall under how to?

Author: Agreed. It could go either way. Reference just felt stronger because while its equal parts "showing" and "informing" its doesn't have the essence of a "hard" how-to, if that makes any sense.

---
# Examples

> For overview examples of the available API tools, take a look at our [wasm examples](../../../../bindings/wasm/examples/node.js). The general API is an abstraction over the rust library, so the examples found [here](../rust/examples.md) still apply (with some minor modifications, see: [api_reference](api_reference.md))

Reviewer: ```suggestion
For overview examples of the available API tools, take a look at our [wasm examples](https://github.com/iotaledger/streams/tree/develop/bindings/wasm/examples). The general API is an abstraction of the rust library, so the [rust examples](../rust/examples.md) still apply with some minor modifications to reflect the differences between the [Rust API](../rust/api_reference.md) and the [Wasm API](api_reference.md).
```
